tripFrontend/src/features/auth/components/CheckBox/CheckBox.tsx

```
@@ -0,0 +1,29 @@
+import type React from 'react';
+
+import { JSX } from 'react';
+
+import { cn } from '@/lib/cn';
+
+type Props = {
+	className?: string;
+	value?: React.RefObject<HTMLInputElement | null>;
+	label?: string;
+	[key: string]: string | object | undefined | Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)